kotsadm/pkg/store/ocistore/license_store.go

```diff
@@ -39,3 +39,34 @@ func (s OCIStore) GetLicenseForAppVersion(appID string, sequence int64) (*kotsv1
 
 	return appVersion.KOTSKinds.License, nil
 }
+
+func (s OCIStore) GetAppSlugForLicense(license string) (string, error) {
+	appSlug, err := s.GetAppSlugForLicense(license)
```

Comment:
i don't understand this, why calling the same function again, and twice? won't this be stuck on a loop? you could just use this for now
```
return nil, ErrNotImplemented
```